libs/cart/src/drivers/magento/queries/fragments/cart-item.ts

```diff
@@ -8,7 +8,7 @@ export const cartItemFragment = gql`
   fragment cartItem on CartItemInterface {
     id
     product {
-      ...product
+      ...magentoProduct
```

Comment:
Let's remove this from this PR, its outside scope.

Reply:
removed